graphql/validate.lua

```diff
@@ -73,6 +141,22 @@ local visitors = {
     rules = { rules.unambiguousSelections }
   },
 
+  -- <field>
+  --
+  -- {
+  --   kind = 'field',
+  --   selectionSet = <selectionSet> (optional),
+  --   alias = <alias> (optional),
+  --   arguments = list of <argument> (optional),
+  --   directives = list of <directive> (optional),
```

Comment:
I suppose, required `name` is missing here

Reply:
Looks correct. Fixed.